db.py

```diff
@@ -151,3 +151,12 @@ def cullOldCodes():
 	DB_CONN.commit()
 	return cursor.rowcount
 
+def getNumCodesWithLength(length):
+	'''Returns the number of codes with the given length'''
+	global DB_CONN
```

Comment:
This will need to be replaced with a getDb() if we want it to work in the state of the develop branch.